### DISCUSSION TITLE
Adding Models to the Glossary - II

### DISCUSSION THREAD

**Discussion by jsvgoncalves** [Depart-de-Sentier/sentier.dev#26] [Workshop Item]
**Goal:** Add a pre-existing (either from model candidates or from previous workshop item) model to the glossary. 

**Key Objectives:**
1. **Model Selection:** Evaluate and select suitable LCI models from the candidate list based on relevance to the glossary's scope, data quality, and technical compatibility.
2. **Compatibility Assessment:** Determine the technical requirements (e.g., data format, software dependencies, licensing) for integrating each selected model into the glossary pipeline.
3. **Integration Strategy:** Develop a standardized process for incorporating LCI models, ensuring integration with the existing glossary structure and data schema.
4. **Model Validity:** Ensure that the model isn't run outside of its boundary conditions and constraints. This should be a first-class concept and users should not have to read the model's metadata, but rather automatically make use of model assumptions, limitations.

**Potential Outcomes:**
- **Second glossary use case:** The glossary will be used to store information about a given model and to store information about a model's IO.
- **Improved Data Interoperability:** Integrating models with consistent data formats and terminology will facilitate data exchange and comparison across different LCA studies.

**Candidate Models:**
- [https://github.com/msmasnadi/OPGEEv4](https://github.com/msmasnadi/OPGEEv4)
- [https://onlinelibrary.wiley.com/doi/abs/10.1002/pip.3695](https://onlinelibrary.wiley.com/doi/abs/10.1002/pip.3695)
- [https://github.com/romainsacchi/LCA_WIND_DK/tree/master](https://github.com/romainsacchi/LCA_WIND_DK/tree/master)
- [https://github.com/romainsacchi/carculator](https://github.com/romainsacchi/carculator)
- [https://pubs.acs.org/doi/10.1021/acs.est.3c03190](https://pubs.acs.org/doi/10.1021/acs.est.3c03190)
- [https://www.epa.gov/warm/versions-waste-reduction-model#v16](https://www.epa.gov/warm/versions-waste-reduction-model#v16)
-  [https://www.sciencedirect.com/science/article/abs/pii/S0360132323011046](https://www.sciencedirect.com/science/article/abs/pii/S0360132323011046)

**Comment by tngTUDOR** [comment]
The sample [Wind turbine model](https://github.com/romainsacchi/LCA_WIND_DK) is drafted [here](https://github.com/YannickSchubert/moeschberg_example_models)
